OpenEdX/View/MainScreenView.swift

```diff
@@ -41,7 +41,7 @@ struct MainScreenView: View {
     var body: some View {
         TabView(selection: $selection) {
             ZStack {
-                DiscoveryView(viewModel: Container.shared.resolve(DiscoveryViewModel.self)!)
+                DiscoveryView(viewModel: Container.shared.resolve(DiscoveryViewModel.self)!, router: Container.shared.resolve(DiscoveryRouter.self)!)
```

Comment:
133 Characters line.
We have to limit it to 80-120 characters.

Let's format it to
```
DiscoveryView(
    viewModel: Container.shared.resolve(DiscoveryViewModel.self)!, 
    router: Container.shared.resolve(DiscoveryRouter.self)!
)
```

Reply:
Done

Reply:
We are partially trying to follow this convention
https://google.github.io/swift/#function-calls
`
When a function call is line-wrapped, each argument is written on its own line.
`

I know we have places in the project with different formatting, but I hope to follow this one for all new written code.

Could you please format it to this style?

Reply:
btw, new XCode code formatting `Control + M` also format code this way

Reply:
Thanks for the tip but I guess the way `Control + M` is formatting we are not following that format in the code base. This is how `Control + M` formatted the above lines 
<img width="495" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/5606473/5ca1823a-952f-4d3e-8c87-aba71a1debb9">

Reply:
Yeah, you're right; it's doing a bit too much 😂

Reply:
But just the main point that I wanted to mention is
`When a function call is line-wrapped, each argument is written on its own line.`
@rnr @eyatsenkoperpetio FYI